clusterloader2/testing/load/config.yaml

```diff
@@ -56,11 +76,20 @@
 
 name: load
 namespace:
-  number: {{$namespaces}}
+  number: {{AddInt $namespaces $schedulerThroughputNamespaces}}
```

Comment:
Technically, we should wrap this into if to make this PR noop without setting the additional env variable.

But in practice, we can leave this without additional if I think. The expected value for schedulerThroughputNamespaces is approx 2, so I don't think creating two more empty namespaces in all tests will be an issue.